guides/common/modules/proc_adding-google-gce-details-to-a-compute-profile.adoc

```diff
@@ -1,41 +1,39 @@
-[id="adding-gce-details-to-a-compute-profile_{context}"]
-= Adding Google Compute Engine Details to a Compute Profile
+[id="Adding_Google_GCE_Details_to_a_Compute_Profile_{context}"]
+= Adding Google GCE Details to a Compute Profile
 
-Use this procedure to add GCE hardware settings to a compute profile.
-When you create a host on GCE using this compute profile, these settings are automatically populated.
+Use this procedure to add Google GCE hardware settings to a compute profile.
+When you create a host on Google GCE using this compute profile, these settings are automatically populated.
 
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-gce-details-to-a-compute-profile_{context}[].
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-Adding_Google_GCE_Details_to_a_Compute_Profile_{context}[].
 
 .Procedure
-
 . In the {ProjectWebUI}, navigate to *Infrastructure* > *Compute Profiles*.
 . In the Compute Profiles window, click the name of an existing compute profile, or click *Create Compute Profile*, enter a *Name*, and click *Submit*.
 . Click the name of the GCE compute resource.
 . From the *Machine Type* list, select the machine type to use for provisioning.
 . From the *Image* list, select the image to use for provisioning.
-. From the *Network* list, select the GCE network to use for provisioning.
+. From the *Network* list, select the Google GCE network to use for provisioning.
 . Optional: Select the *Associate Ephemeral External IP* check box to assign a dynamic ephemeral IP address that {Project} uses to communicate with the host.
 This public IP address changes when you reboot the host.
-If you need a permanent IP address, reserve a static public IP address on GCE and attach it to the host.
+If you need a permanent IP address, reserve a static public IP address on Google GCE and attach it to the host.
 . In the *Size (GB)* field, enter the size of the storage to create on the host.
 . Click *Submit* to save the compute profile.
 
-[id="cli-adding-gce-details-to-a-compute-profile_{context}"]
+[id="cli-Adding_Google_GCE_Details_to_a_Compute_Profile_{context}"]
 .CLI procedure
-
-. Create a compute profile to use with the GCE compute resource:
+. Create a compute profile to use with the Google GCE compute resource:
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer compute-profile create --name _gce_profile_
+# hammer compute-profile create --name _My_GCE_Compute_Profile_
 ----
-
 . Add GCE details to the compute profile.
```

Comment:
Replace the trailing period with a colon.

Reply:
Applied.